pallas-codec/src/flat/encode/encoder.rs

```diff
@@ -237,13 +250,13 @@ impl Encoder {
                 self.used_bits += num_bits;
                 let unused_bits = 8 - self.used_bits;
                 match unused_bits {
-                    x if x > 0 => {
-                        self.current_byte |= val << x;
-                    }
-                    x if x == 0 => {
+                    0 => {
                         self.current_byte |= val;
                         self.next_word();
                     }
+                    x if x > 0 => {
+                        self.current_byte |= val << x;
+                    }
```

Comment:
@rvcas this change was suggested by clippy. Seems harmless, please review.

Reply:
Yea totally fine